docs/_connect-with-keycloak/03-add-identity-provider.md

```diff
@@ -65,4 +95,57 @@ This will take you to the AAF Discovery Service (known as AAF Central).
 
 * If authentication is successful, you will be taken back to your Keycloak account console.
 
-![New user account](/assets/images/connect-with-keycloak/new-user-account.png)
\ No newline at end of file
+![New user account](/assets/images/connect-with-keycloak/new-user-account.png)
+
+
+### Skipping Discovery Service
+
+
+AAF's implementation of OpenID Connect allows service administrators to configure their service [to skip the discovery service](/openid-connect-integration/05-skipping-discovery-service) and login directly through a specified IdP. This can also be done through Keycloak.
+
+### Getting started
+
+
+Find the **entityID** of the IdP that you wish to directly login through. This can be done by looking through the AAF metadata or your own registrations.
+
+<a href="https://md.test.aaf.edu.au/" class="btn btn-outline-primary mb-3">AAF Test Metadata</a>
+<br>
+For Example, the AAF Virtual Home is: `https://vho.aaf.edu.au/idp/shibboleth`
```

Comment:
I believe anyone can also view all the IdP info? But best to double check

https://manager.aaf.edu.au/identity_providers